gordon_gcp/schemas/audit-log.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "id": "https://github.com/spotify/gordon-gcp/blob/master/gordon_gcp/schemas/audit-log.schema.json",
+    "title": "Google Audit Log Messages",
+    "description": "Schema for parsing Google Audit Log Messages",
+    "required": ["protoPayload", "logName"],
+    "properties": {
+        "protoPayload": {
+            "type": "object",
+            "properties": {
+                "methodName": {
+                    "type": "string",
+                    "description": "Action performed on a GCP",
+                    "enum": [
+                        "v1.compute.instances.insert",
+                        "v1.compute.instances.delete"
+                    ]
+                },
+                "resourceName": {
+                    "type": "string",
+                    "description": "",
+                    "examples": [
+                        "projects/a-project-id/zones/a-zone-name/instances/an-instance-name"
+                    ],
+                    "pattern": "^projects/[0-9]+/zones/[a-z\\-0-9]+/instances/[a-z\\-0-9]+"
+                }
+            },
+            "required": ["methodName", "resourceName"]
+        },
+        "logName": {
+            "type": "string",
+            "description": "Name of audit log",
+            "examples": [
+                "projects/a-project-id/logs/cloudaudit.googleapis.com%2Factivity"
+            ],
+            "pattern": "^projects/[a-z][a-z\\-0-9]{5,29}/logs/cloudaudit.googleapis.com%2Factivity"
+        },
+        "receiveTimestamp": {
```

Comment:
I figured these two could be useful for idempotence, but I was unsure so I left them as optional.